cos-gpu-installer-docker/entrypoint.sh

```diff
@@ -37,6 +39,12 @@ LOCK_FILE="${ROOT_MOUNT_DIR}/tmp/cos_gpu_installer_lock"
 LOCK_FILE_FD=20
 set +x
 
+TOOLCHAIN_DOWNLOAD_URL=""
```

Comment:
Can you please add a comment to explain that these variables are expected to be set by set_compilation_env() ?